### PR TITLE
Release 2.5.0

### PR DESCRIPTION
**Pull request by YuyaoHu**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,12 +17,30 @@
 
 * **[Behavior change]** Fix a security issue with the `Exception` field on `ErrorReport` objects. As a result, the `Exception.StackTrace` now holds the raw stack trace, and the `Exception.Message` field is `null`.
 
+#### Uwp
```
Comment (Cloudwu1) [inline CHANGELOG.md]: It should be UWP.

File: RELEASE.md [hunk 2]
```diff
@@ -1,3 +1,44 @@
...
+
+* **[Fix]** Fix `AppCenterStarter.m` was not included in the first build.
+
+### App Center Crashes
+
+#### Android
+
+* **[Behavior change]** Fix a security issue with the `Exception` field on `ErrorReport` objects. As a result, the `Exception.StackTrace` now holds the raw stack trace, and the `Exception.Message` field is `null`.
+
+#### Uwp
```
Comment (Cloudwu1) [inline RELEASE.md]: The same

**Review by Cloudwu1 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (YuyaoHu) [reply CHANGELOG.md]: Okay, I've made the changes.

File: CHANGELOG.md [hunk 3]
```diff
@@ -12,12 +17,30 @@
 
 * **[Behavior change]** Fix a security issue with the `Exception` field on `ErrorReport` objects. As a result, the `Exception.StackTrace` now holds the raw stack trace, and the `Exception.Message` field is `null`.
 
+#### UWP
+
+* **[Feature]** App Center now supports crashes for sideloaded UWP applications.
+
+* **[Feature]** APIs in the Crashes module are now implemented for UWP: handled errors, crash attachments, crash callbacks, getting crash information about last session, and enabling/disabling the module. Detecting low memory warning is not supported.
+
```
Comment (Cloudwu1) [inline CHANGELOG.md]: No need blank line.

Comment (YuyaoHu) [reply CHANGELOG.md]: Okay

**Review by Cloudwu1 — APPROVED** [review]

**Review by annakocheshkova — APPROVED** [review]

**Pull request merged**